### PR TITLE
Add translit ipa

### PR DESCRIPTION
**Pull request by ManjotS**

### CODE REVIEW

**Review by tarunsingh5 — APPROVED** [review]

**Comment by tarunsingh5** [comment]
@ManjotS @navdeepsinghkhalsa can we merge & close this out

**Comment by ManjotS** [comment]
@navdeepsinghkhalsa I  am confident enough in it (it will need adjustments in the future just   as  english does)

can we merge?

File: .eslintignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 dist/*
 coverage
+anvaadtest.js
```
Comment (navdeepsinghkhalsa) [inline .eslintignore]: It might be best to keep this change and in `.gitignore` local

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "anvaad-js",
-  "version": "1.0.9",
```
Comment (navdeepsinghkhalsa) [inline package.json:3]: This will be done by `npm version` when releasing

File: src/translit_modules/ipa.js [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+/**
+ * Returns a transliteration of Gurmukhi script
+ *
+ * @since 1.0.0
+ * @param {string} gurmukhi The string from to generate transliteration
+ * @returns {string} Returns a string of text
+ * @example
+ *
+ * translit('Awie imlu gurisK Awie imlu qU myry gurU ky ipAwry ]');
+ * // => 'aai mil gurasikh aai mil too mayray guroo kay piaaaray ||'
```
Comment (navdeepsinghkhalsa) [inline src/translit_modules/ipa.js]: Could you please update the expected result?

File: src/translit_modules/ipa.js [hunk 4]
```diff
@@ -0,0 +1,281 @@
+/* eslint no-useless-escape: 0 */
+
+const unescape = require('lodash.unescape');
+
+/**
+ * Returns a transliteration of Gurmukhi script
+ *
+ * @since 1.0.0
```
Comment (navdeepsinghkhalsa) [inline src/translit_modules/ipa.js]: This should be 1.1.0

**Review by navdeepsinghkhalsa — CHANGES_REQUESTED** [review]

**Comment by navdeepsinghkhalsa** [comment]
Could you also please resolve conflicts?

File: src/__tests__/translit.test.js [hunk 5]
```diff
@@ -3,6 +3,9 @@ const translit = require('../translit');
 describe('translit', () => {
   it('Should return transliteration of Gurmukhi', () => {
     expect(JSON.stringify(translit('CMqu ] ijau jwnhu iqau rwKu hir pRB qyirAw ] kyqy gnau AsMK Avgx myirAw ] AsMK Avgx Kqy Pyry inqpRiq sd BUlIAY ] moh mgn ibkrwl mwieAw qau pRswdI GUlIAY ] lUk krq ibkwr ibKVy pRB nyr hU qy nyirAw ] ibnvMiq nwnk dieAw Dwrhu kwiF Bvjl PyirAw ]1]', 'all')))
-      .toBe(JSON.stringify({ english: 'chha(n)t || jiau jaanahu tiau raakh har prabh teriaa || kete ganau asa(n)kh avagan meriaa || asa(n)kh avagan khate fere nitaprat sadh bhooleeaai || moh magan bikaraal maiaa tau prasaadhee ghooleeaai || look karat bikaar bikhaRe prabh ner hoo te neriaa || binava(n)t naanak dhiaa dhaarahu kaadd bhavajal feriaa ||1||' }));
+      .toBe(JSON.stringify({
```
Comment (navdeepsinghkhalsa) [inline src/__tests__/translit.test.js:6]: Is one test enough?

File: src/translit_modules/ipa.js (hunk 3 above)
Comment (tarunsingh5) [reply src/translit_modules/ipa.js]: @ManjotS

**Comment by tarunsingh5** [comment]
@ManjotS please follow up here

**Review by navdeepsinghkhalsa — APPROVED** [review]

**Comment by ManjotS** [comment]
This was actually a part of Guaravjeet Singh's work...

**Pull request merged**